README.md

```diff
@@ -1,5 +1,54 @@
+[![MoreCompany Discord](https://img.shields.io/badge/MoreCompany%20Discord-5865F2?style=for-the-badge)](https://discord.gg/cKa6sPBFZ9) [![GitHub Issue Tracker](https://img.shields.io/badge/Issue%20Tracker-2dba4e?style=for-the-badge)](https://github.com/notnotnotswipez/MoreCompany/issues)
 # MoreCompany
-Hello,
-this mod is not affiliated with BiggerLobby.
 
-Although the license of the mod is MIT, it would be really silly to take all of my patches and put them in another mod or just reupload a fork. I can't do anything legally about it but yknow, please?
+MoreCompany expands Lethal Company's max player count for larger player lobbies and adds new cosmetics, all in a stable experience. Some UI elements may not fully support more than 8 players. Available to download from [Thunderstore](https://thunderstore.io/c/lethal-company/p/notnotnotswipez/MoreCompany/) or view the source code on [GitHub](https://github.com/notnotnotswipez/MoreCompany).
+
+## Release Trailer
+[![](http://img.youtube.com/vi/yHxU4YrA8Ag/0.jpg)](http://www.youtube.com/watch?v=yHxU4YrA8Ag "MoreCompany Launch Trailer on YouTube")
+
+## Features
+- Cosmetics for players to choose in menu and wear in-game, and the ability to create your own (just check out the [guide](https://github.com/notnotnotswipez/MoreCompany/wiki/Cosmetic-Creation))
+- Scavenge results modified to fit 8 players and will show their status at the end of the round.
+- Spectators screen properly fits more than 4 players, and will not go offscreen.
+- The player volume menu accounts for more than 4 players, by adding a scrollbar to the UI.
+
+## Issues
+If you encounter any issues with the mod, please report them on the [MoreCompany Discord](https://discord.gg/cKa6sPBFZ9) or via the [GitHub Issue Tracker](https://github.com/notnotnotswipez/MoreCompany/issues). Please ensure you check for known issues on the [GitHub Issue Tracker](https://github.com/notnotnotswipez/MoreCompany/issues) or [`CHANGELOG.md`](https://github.com/notnotnotswipez/MoreCompany/blob/master/CHANGELOG.md) BEFORE reporting.
+
+# Why choose MoreCompany?
+Although there is another mod that does a very similar thing, it has a few stability issues regarding the functionality of the game. In the development of MoreCompany, I had stability and polish in mind, I wanted to make a relatively frictionless experience regarding expanded player counts.
+
+In my testing of an 8 player lobby, I experienced no issues nor did anyone else in the lobby. A lot of game conditions were tested, but there may be conditions which were not tested that may be broken or cause a softlock.
+
+This mod is not affiliated with [BiggerLobby](https://thunderstore.io/c/lethal-company/p/bizzlemip/BiggerLobby/)/[LethalPlayers](https://thunderstore.io/c/lethal-company/p/Lightmaxifrvr/LethalPlayers/) nor does it use any code from that mod! Any similarities are purely coincidental.
+
+# Installation
+MoreCompany **must** be installed on both the server / host and all connecting clients.
```

Comment:
This will be incorrect if/whenever https://github.com/notnotnotswipez/MoreCompany/pull/96 goes in since that PR makes it so:
- Vanilla players can join MoreCompany lobbies if the crew limit is 4 or less (if they are invited or if they search for the "morecompany" tag on the lobby list)
- MoreCompany players can join vanilla lobbies (if they are invited or if they search for the "none" tag on the lobby list)

Reply:
I'll leave it for now as I'm not sure when it might get merged as to not require that merge before this one just in case it takes a while for that pull to be ready.